### PR TITLE
[zh-cn]: update translation of Location `toString()` method

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/location/tostring/index.md [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+title: Location：toString() 方法
 slug: Web/API/Location/toString
+l10n:
+  sourceCommit: 1238ffad886924b20549d0cf3adca735cb0d074f
 ---
 
 {{ApiRef("Location")}}
 
-**`toString()`**{{domxref("Location")}}接口的 stringifier 方法返回包含整个 URL 的{{domxref("USVString")}}。它是{{domxref("Location.href")}}的只读版本。
+{{domxref("Location")}} 接口的 **`toString()`** {{Glossary("stringifier", "字符串化")}}方法返回一个包含整个 URL 的字符串。他是 {{domxref("Location.href")}} 的只读版本。
```
Comment (T34-active) [inline files/zh-cn/web/api/location/tostring/index.md:10]:
```suggestion
{{domxref("Location")}} 接口的 **`toString()`** {{Glossary("stringifier", "字符串化")}}方法返回一个包含整个 URL 的字符串。它是 {{domxref("Location.href")}} 的只读版本。
```

File: files/zh-cn/web/api/location/tostring/index.md [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
 ```
 
-## 例子
+### 参数
+
+无。
+
+### 返回值
+
+表示 URL 对象的字符串。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/location/tostring/index.md:24]:
```suggestion
表示对象的 URL 的字符串。
```

**Review by jasonren0403 — COMMENTED** [review]
nit

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**